packages/stacking/README.md

```diff
@@ -0,0 +1,94 @@
+# @stacks/stacking [![npm](https://img.shields.io/npm/v/@stacks/stacking?color=red)](https://www.npmjs.com/package/@stacks/stacking)
+Library for PoX Stacking.
+
+## Installation
+
+```
+npm install @stacks/stacking
+```
+
+## Initialization
+```typescript
+const { StacksTestnet } = require('@stacks/network');
+const { Stacker } = require('@stacks/stacking');
```

Comment:
this is very likely out of scope, but it might be beneficial to think about it before we work on it: how would delegation impact the interface?

Reply:
Yep, I can see it as an additional method in the class for delegation.